### DISCUSSION TITLE
If there is a option to change the splash screen or just disable it would be splendid. 😭

### DISCUSSION THREAD

**Discussion by hdchieh** [DavidS95/Smokeless_UMAF#7] [Q&A] [answered]
I only want to get rid of the splash screen, but my manufacturer has not released any BIOS files but a specialized tool to update the  BIOS.  It would be much easier if this universal tool could disable the BIOS logo.

**Comment by hdchieh** [comment]
wondering if it would be possible to have a "Full screen logo show" or "Quiet boot" option added to your project. Greatly appreciate!

**Comment by DavidS95** [comment, marked as answer]
This tool simply reveals hidden bios settings and allows you to edit them, if the option does not exist it cannot be force to appear.
